build.mjs

```diff
@@ -12,7 +12,21 @@ const SRC_DIR = path.join("src", "js");
 const TESTS_DIR = "tests";
 const OUT_DIR = "bundle";
 const SHIMS = new Map();
-const TEST_FILES = await fs.readdir(TESTS_DIR);
+const TEST_FILES = [];
+async function readAllTestFilesRecursively(directory) {
```

Comment:
We can refactor this to a `readDirRecursive` function that takes in a directory and a predicate filter function as argument. We can also use `readDir` option `withFileTypes` that saves the additional `fs.stat` call.